packages/mui-nested-menu/src/components/IconMenuItem.tsx

```diff
@@ -3,6 +3,7 @@ import MenuItem, { MenuItemProps } from '@mui/material/MenuItem';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/system/Box';
 import styled from '@mui/material/styles/styled';
+import { SxProps } from '@mui/system';
```

Comment:
Can this be changed to full path import following this 
PR: https://github.com/steviebaa/mui-nested-menu/pull/18

Reply:
Changed to `'@mui/system/styleFunctionSx'`